tests/format_sql/conftest.py

```diff
@@ -16,16 +16,15 @@ def pytest_collect_file(parent, path):
     """Collect format_sql tests."""
     if path.basename == "expect.sql":
         print(path)
-        return FormatTest(path, parent)
+        return FormatTest.from_parent(parent, fspath=path)
```

Comment:
Directly instantiating test classes was deprecated in favor of `.from_parent` in pytest 5.4, and removed in pytest 6